services/src/handlers/ebv.rs

```diff
@@ -465,20 +475,28 @@ impl<C: Context> Task<C::TaskContext> for EvbMultiOverviewTask<C> {
                             // must not happen, since we spawned a task that only returns a `NetCdfCfOverviewResponse`
                         }
                     }
-                    TaskStatus::Failed { error } => {
+                    TaskStatus::Aborted { .. } => {
+                        if log_enabled!(Debug) {
```

Comment:
If I read the description of `log_enabled` right you don't have to use that here because it is not expensive to compute a static str

Reply:
[fa6ef67](https://github.com/geo-engine/geoengine/pull/586/commits/fa6ef67f578d959a0a24309522e8ad3f0752b58f)